.github/workflows/dependency_review.yml

```diff
@@ -0,0 +1,19 @@
+name: Deriv-app Dependency Review
+on: 
+  pull_request:
+    branches:
+        - master
+jobs:
+    dependency_review:
+        runs-on: ubuntu-latest
+        permissions:
+            contents: read
+            pull-requests: write
```

Comment:
do we need the pull-requests permission?

Reply:
Yes @balakrishna-deriv, We need them in order to write the summary on the `PR`.